### ISSUE TITLE
New Practice: Used Linter for Ruby

### ISSUE THREAD

**Issue by prokopsimek** [DXHeroes/dx-scanner#416]
## New Practice Proposal

Create a new practice that checks if is used linter for Ruby.

**Comment by avats-dev** [comment]
Hey @prokopsimek , I want to work on this. Can you guide me a bit how to resolve this? Thanks.

**Comment by adelkahomolova** [comment]
Hi @avats-dev . It should do the same thing as the practice for Javascript [ESLintUsedPractice](https://github.com/DXHeroes/dx-scanner/blob/master/src/practices/JavaScript/ESLintUsedPractice.ts) just for Ruby. So choose the most widely used linter for Ruby and do almost the same code as we used for JS. Is it clearer now?

**Comment by avats-dev** [comment]
Yeah, I'm seeing other practices and will update soon. Thanks for the insights @adelkahomolova .

**Comment by adelkahomolova** [comment]
Hi @avats-dev how is it going?

**Comment by adelkahomolova** [comment]
Hi, @avats-dev it's quite a long time since our last contact. Is there any progress?
